### DISCUSSION TITLE
How to install Dokku on Ubuntu 23.04 with Ansible?

### DISCUSSION THREAD

**Discussion by bfontaine** [dokku/dokku#6531] [Q&A] [answered]
Hello,
I’m trying to install Dokku with Ansible, but [the official repo](https://github.com/dokku/ansible-dokku?tab=readme-ov-file#platform-requirements) doesn’t support Ubuntu 23.04, probably because there are no packages for it (#6340). Is the support of this version planned anytime soon? Can I help?
Thanks in advance

**Comment by bfontaine** [comment, marked as answer]
In the end I just reset my VPS to 22.04.

**Reply by josegonzalez** [reply]
For a variety of reasons - mostly time - the project doesn't support non-LTS installs for Dokku installations. Since your app lives in a container and the tools we use are updated fairly often, using an LTS release is the safest path forward.
